Resources/config/services.yml

```diff
@@ -31,3 +32,16 @@ services:
         class: %support_tools.info_collectors.php.ezc.class%
         arguments:
             - @support_tools.info_collector.system_info.ezc
+
+    support_tools.info_collectors.symfony.kernel.config:
+        class: %support_tools.info_collectors.symfony.kernel.config.class%
+        arguments:
+            - %kernel.environment%
```

Comment:
Ok, I can inject `@kernel` instead. Need to do more mocking in the test before I push it. I'd like to keep the `%kernel.bundles%` parameter, to avoid complicating the collector (I'd rather use the hash it provides directly, than build the same thing in collector code).